resources/views/components/base/form-field.blade.php

```diff
@@ -4,6 +4,7 @@
     'id' => null,
     'inline' => false,
     'inputType' => null,
+    'isLabelVisible' => true,
```

Comment:
There should always be a label for non-hidden form fields, but it's also desirable to be able to manipulate the visibility of the label. Setting this prop to `false` puts the label in `sr-only` mode.

Reply:
Do we have use-cases where we actually want to hide the labels?
The way it is written now the label _has_ to be visible because the inputs are not meant to be used without them. I'm still not fully convinced that we should allow for labels to be hidden like that.

Reply:
note that to me all the forms that ever should exist for users already exist and the all require labels. i'd like to have most of the dev/mod dropdowns to migrated out of the frontend eventually which reduces the need for custom form components.

Reply:
just realized that it's about the reply textarea label - i left that visible on purpose. hiding it like this makes sense if that's preferred 👍

Reply:
Yep, you nailed it 👍 This all comes down to the reply textarea label, which uniquely makes sense not to be visible.